docs/02-app/01-building-your-application/07-configuring/05-mdx.mdx

```diff
@@ -254,16 +254,15 @@ import MdxLayout from '../components/mdx-layout'
 # Welcome to my MDX page!
 
 export default function MDXPage({ children }) {
-  return <MdxLayout>{children}</MdxLayout>;
-
+  return <MdxLayout>{children}</MdxLayout>
 }
 ```
 
 </PagesOnly>
 
 ## Remark and Rehype Plugins
 
-You can optionally provide `remark` and `rehype` plugins to transform the MDX content.
+You can optionally provide `remark` and `rehype` plugins head to the MDX content.
```

Comment:
@Pyr33x Not sure about this change! Can you clarify what you meant with "head" here?

Reply:
I just meant if we can use it instead of "transform to", if you think it's not reasonable enough, then I'll revert it

Reply:
@Pyr33x Noted! Let's revert to the original for this change.

Reply:
I'm done!